src/util/hash-utils.ts

```diff
@@ -0,0 +1,9 @@
+import { createHash } from "crypto";
```

Comment:
I was wondering if we maybe should put it to the existing file with out hashing function? https://github.com/atlassian/github-for-jira/blob/eeeff2f86a9c94d8dc8baf0dcaed42273f1d074f/src/util/encryption.ts#L4-L4 

There we have `createHashWithSharedSecret` , should we have some `createHashWithoutSharedSecret`, to 
1. keep things uniform
2. explicitly separate those

wdyt?

Reply:
Thought of that. But the folder name is `encryption`, but the hash is not about encryption. Still a bit inconsistent.

Reply:
Well, sha256 is all all about the [encryption/cryptography](https://en.wikipedia.org/wiki/Cryptographic_hash_function) (as opposed to data structures' one [that is used in hash table](https://en.wikipedia.org/wiki/Hash_function))...  IMHO putting it into encryption utils makes more sense tbh...

Reply:
Maybe we can rename the folder name in another PR. `Encryption` should means you can `decrypt` it later. But right now all it has are just `hash`, which we can't un-hash.
Anyway I've taken your advice and put it into that folder.